conf/pvers.c

```diff
@@ -804,6 +804,22 @@ static int conf_pver_virtual_create(const struct m0_fid *fid,
 	return rc;
 }
 
+/**
+ * Check if failures at any level has reached max allowed failures.
+ */
+static bool has_reached_max_failure(struct m0_conf_pver *pv,
+				    const uint32_t      *srecd)
+{
+	int i = 0;
+
+	while(i < M0_CONF_PVER_HEIGHT) {
+		if (srecd[i] == pv->pv_u.subtree.pvs_tolerance[i])
```

Comment:
In our development environment, often we do not set any tolerance vectors leaving them to be '0'. This function will return true in such cases. Do you think we should check for pvs_tolerance[i] > 0 condition before comparing it with srecd[i]?
Also, is there a possibility that srecd[i] is greater than pvs_tolerance[i]? If yes, you may have to check for it and the calling code may also need to change accordingly.